connector/src/main/java/com/google/cloud/bigquery/connector/common/TableStatistics.java

```diff
@@ -0,0 +1,28 @@
+package com.google.cloud.bigquery.connector.common;
+
+import java.util.Optional;
+import java.util.OptionalLong;
+
+public class TableStatistics {
```

Comment:
docs.  Also it seems that there is TableStatistics and EstimatedTableStatistics, is there some way to change the naming to differentiate these a bit more?